### ISSUE TITLE
Запятые в неположенном месте, потенциальная ошибка

### ISSUE THREAD

**Issue by MrNovado** [MinakovaLisa/Labyrinth#4]
https://github.com/MinakovaLisa/Labyrinth/blob/6f25a5874731151ffb4f91a0390843c507874796/components/navigator.js#L9-L23

Здесь не должно быть запятых, - по спецификации тут могут (или могут не быть) точки с замятой `;`.
Запятая - это довольно опасный оператор, пропускающий и перечисляющий действия, - используй его очень аккуратно.

**Comment by liza-rd-brain** [comment]
так-то же это просто перечисление аргументов?! и тут должны быть запятые?!
